libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/model/TurnIconResources.kt

```diff
@@ -0,0 +1,1685 @@
+            .turnIconOffRamp(turnIconOffRamp)
+            .turnIconOffRampLeft(turnIconOffRampLeft)
+            .turnIconOffRampRight(turnIconOffRampRight)
+            .turnIconOffRampSlightLeft(turnIconOffRampSlightLeft)
+            .turnIconOffRampSlightRight(turnIconOffRampSlightRight)
+            .turnIconOnRamp(turnIconOnRamp)
+            .turnIconOnRampLeft(turnIconOnRampLeft)
+            .turnIconOnRampRight(turnIconOnRampRight)
+            .turnIconOnRampStraight(turnIconOnRampStraight)
+            .turnIconOnRampSlightLeft(turnIconOnRampSlightLeft)
+            .turnIconOnRampSlightRight(turnIconOnRampSlightRight)
+            .turnIconOnRampSharpLeft(turnIconOnRampSharpLeft)
+            .turnIconOnRampSharpRight(turnIconOnRampSharpRight)
+            .turnIconRamp(turnIconRamp)
+            .turnIconRotary(turnIconRotary)
+            .turnIconRotaryLeft(turnIconRotaryLeft)
+            .turnIconRotaryRight(turnIconRotaryRight)
+            .turnIconRotaryStraight(turnIconRotaryStraight)
+            .turnIconRotarySlightLeft(turnIconRotarySlightLeft)
+            .turnIconRotarySlightRight(turnIconRotarySlightRight)
+            .turnIconRotarySharpLeft(turnIconRotarySharpLeft)
+            .turnIconRotarySharpRight(turnIconRotarySharpRight)
+            .turnIconRoundabout(turnIconRoundabout)
+            .turnIconRoundaboutLeft(turnIconRoundaboutLeft)
+            .turnIconRoundaboutRight(turnIconRoundaboutRight)
+            .turnIconRoundaboutStraight(turnIconRoundaboutStraight)
+            .turnIconRoundaboutSlightLeft(turnIconRoundaboutSlightLeft)
+            .turnIconRoundaboutSlightRight(turnIconRoundaboutSlightRight)
+            .turnIconRoundaboutSharpLeft(turnIconRoundaboutSharpLeft)
+            .turnIconRoundaboutSharpRight(turnIconRoundaboutSharpRight)
+            .turnIconTrafficCircle(turnIconTrafficCircle)
+            .turnIconTrafficCircleLeft(turnIconTrafficCircleLeft)
+            .turnIconTrafficCircleRight(turnIconTrafficCircleRight)
+            .turnIconTrafficCircleSlightLeft(turnIconTrafficCircleSlightLeft)
+            .turnIconTrafficCircleSlightRight(turnIconTrafficCircleSlightRight)
+            .turnIconTurnLeft(turnIconTurnLeft)
+            .turnIconTurnRight(turnIconTurnRight)
+            .turnIconTurnStraight(turnIconTurnStraight)
+            .turnIconTurnSlightLeft(turnIconTurnSlightLeft)
+            .turnIconTurnSlightRight(turnIconTurnSlightRight)
+            .turnIconTurnSharpLeft(turnIconTurnSharpLeft)
+            .turnIconTurnSharpRight(turnIconTurnSharpRight)
+            .turnIconUpdown(turnIconUpdown)
+            .turnIconUturn(turnIconUturn)
+    }
+
+    /**
+     * Returns a string representation of the object.
+     */
+    override fun toString(): String {
+        return "TurnIconResources(" +
+            "turnIconArrive=$turnIconArrive)" +
+            "turnIconArriveLeft=$turnIconArriveLeft" +
```

Comment:
```suggestion
            "turnIconArriveLeft=$turnIconArriveLeft, " +
```

We're missing this for all the fields below 👇